sdk/spring/CHANGELOG.md

```diff
@@ -83,12 +128,30 @@ This section includes changes in `spring-cloud-azure-starter-eventhubs` module.
 - Change class from `com.azure.spring.messaging.PartitionSupplier` to `com.azure.spring.messaging.eventhubs.core.PartitionSupplier` [#27422](https://github.com/Azure/azure-sdk-for-java/issues/27422).
 - Remove parameter of `PartitionSupplier` from the sending API for a single message in `EventHubsTemplate`. 
 Please use message headers of `com.azure.spring.messaging.AzureHeaders.PARTITION_ID` and `com.azure.spring.messaging.AzureHeaders.PARTITION_KEY` instead [#27422](https://github.com/Azure/azure-sdk-for-java/issues/27422).
+- Change the message header prefix from `azure_eventhub` to `azure_eventhubs_` [#27746](https://github.com/Azure/azure-sdk-for-java/pull/27746).
```

Comment:
same here, these changes should either be in spring-integration-eventhubs, spring-integration-eventhubs, or spring-cloud-azure-autoconfigure